docs/examples/AsyncCallbacks.js

```diff
@@ -19,10 +19,11 @@ const loadOptions = (inputValue, callback) => {
   }, 1000);
 };
 
-export default class WithCallbacks extends Component<*, State> {
+export default class WithCallbacks extends Component<State> {
   state = { inputValue: '' };
   handleInputChange = (newValue: string) => {
-    const inputValue = newValue.replace(/\W/g, '');
+    //const inputValue = newValue.replace(/\W/g, '');
```

Comment:
Remove?

Reply:
for a special character, so we can enter them same as in promise